.gitignore

```diff
@@ -7,10 +7,13 @@ node_modules
 coverage
 
 # production
-build
+/build
```

Comment:
I forgot, how are absolute looking paths handled in gitignore again?

Reply:
There's a folder called "build" inside the generated semantic ui content that we don't want to be ignored, so I changed this to an absolute path so we would only ignore this build folder at the top level.

Reply:
👌